ResearchSuiteUI/ResearchSuiteUI/iOS/Step View Controllers/RSDTableStepViewController.swift

```diff
@@ -404,8 +436,7 @@ open class RSDGenericStepViewController: RSDStepViewController, UITableViewDataS
     ///     - indexPath: The given index path.
     /// - returns: The table view cell configured for this index path.
     open func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let identifier = "\(indexPath.section)-\(indexPath.row)"
-        let cell = dequeueCell(in: tableView, at: indexPath) ?? UITableViewCell(style: .default, reuseIdentifier: identifier)
+        let cell = dequeueCell(in: tableView, for: indexPath) ?? UITableViewCell(style: .default, reuseIdentifier: "__BasicCell")
```

Comment:
GitHub won't let me comment on the doc-comment description above directly, but is it still true that the default implementation uses a unique reuse identifier? Looks like it's now the same for all index paths?